src/ducks/sharing/autosuggest.styl

```diff
@@ -0,0 +1,43 @@
+@import 'cozy-ui'
+
+.container
+    position relative
+
+.suggestionsContainer
+    position absolute
+    top 100%
+    margin-top 1px
+    z-index 500
+    width 100%
+    max-height em(170px)
+    overflow auto
+    border 1px solid silver
+    border-radius 4px
+    background white
+    display none
+
+.suggestionsContainerOpen
+    display block
+
+.suggestionsList
```

Comment:
The list is generated by the [Autosuggest](https://github.com/moroshko/react-autosuggest) component ;)